### PR TITLE
Stop passing peers to delegate actions - Closes #1148 

### LINKED ISSUE #1148: Stop passing peers to delegate actions

### PR DESCRIPTION
**Pull request by michaeltomasik**
### What was the problem?
#1148 
### Review checklist
- The PR solves #1148 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

**Comment by slaweet** [comment]
@michaeltomasik you mean line 93, I mean line 63 in votingListView with `votesFetched`

**Comment by slaweet** [comment]
Since this is the last PR of removing active peers, there should be no peers and activePeer left in `src/components`.
There is still a lot:
```
■■■■[slaweet:~/git/hub-review/src/components] $
$ git grep 'activePeer:'
accountTransactions/index.js:  activePeer: state.peers.data,
authenticate/authenticate.js:      activePeer: this.props.peers.data,
authenticate/authenticate.test.js:      activePeer: props.peers.data,
authenticate/authenticate.test.js:      activePeer: props.peers.data,
confirmVotes/confirmVotes.test.js:    activePeer: peers.data,
confirmVotes/index.js:  activePeer: state.peers.data,
confirmVotes/index.test.js:    //   activePeer: peers,
searchBar/index.js:  activePeer: state.peers.data,
sendReadable/send.test.js:    activePeer: {},
sendWritable/index.js:  activePeer: state.peers.data,
sendWritable/send.test.js:      activePeer: {},
transactions/explorerTransactions/index.js:  activePeer: state.peers.data,
transactions/explorerTransactions/index.test.js:      activePeer: match.any,
transactions/explorerTransactions/index.test.js:      activePeer: match.any,
transactions/walletTransactions/index.test.js:      activePeer: match.any,
transactions/walletTransactions/index.test.js:      activePeer: match.any,
voteUrlProcessor/index.js:  activePeer: state.peers.data,
votingListView/votingListView.test.js:    activePeer: {},

■■■■[slaweet:~/git/hub-review/src/components] $
$ git grep 'peers:'
account/account.test.js:      peers: {
account/index.js:  peers: state.peers,
accountTransactions/index.js:  peers: state.peers,
accountTransactions/index.test.js:    peers: peersReducer,
app/index.test.js:      peers: { data: { options: {} }, options: {} },
app/index.test.js:      peers: {
authenticate/index.js:  peers: state.peers,
header/header.test.js:    peers: { data: {}, options: {} },
header/index.js:  peers: state.peers,
header/index.test.js:    peers: { data: {}, options: {} },
loadingBar/index.js:  peers: state.peers,
loadingBar/loadingBar.test.js:    peers: {},
loadingBar/loadingBar.test.js:    wrapper.setProps({ peers: { data: {} } });
login/index.js:  peers: state.peers,
registerDelegate/index.js:  peers: state.peers,
singleTransaction/index.js:  peers: state.peers,
transactions/explorerTransactions/index.test.js:    peers: peersReducer,
transactions/transactionDetailView.js:  peers: state.peers,
transactions/transactionDetailView.test.js:        peers: {
transactions/transactionDetailView.test.js:        peers: {
transactions/transactionList.test.js:    peers: peersReducer,
transactions/transactionOverview.js:  peers: state.peers,
transactions/transactionOverview.test.js:      peers: {
transactions/transactionOverview.test.js:      peers: { status: {}, options: {}, data: {} },
transactions/transactionType.test.js:  peers: {
transactions/walletTransactions/index.test.js:    peers: peersReducer,
voteUrlProcessor/index.test.js:    peers: { data: {} },
voting/index.test.js:      peers: {},
voting/voting.test.js:    peers: peersReducer,
votingListView/votingHeader.test.js:    peers: {
```
Maybe some of them need to stay, but most shouldn't need to.

File: src/actions/voting.test.js [hunk 1]
```diff
@@ -213,7 +214,7 @@ describe('actions: voting', () => {
       peers: {
         data: {
           options: {
-            name: 'Mainnet',
+            name: Networks.mainnet.name,
```
Comment (faival) [inline src/actions/voting.test.js]: I would use only capital letters to name a variable if and only if its a react component or a class.

**Comment by slaweet** [comment]
The list is now shorter but there's still a lot of peers and activePeer:
```
■■■■[slaweet:~/git/hub-review/src/components] $
$ git grep 'activePeer:'
transactions/explorerTransactions/index.test.js:      activePeer: match.any,
transactions/explorerTransactions/index.test.js:      activePeer: match.any,
transactions/walletTransactions/index.test.js:      activePeer: match.any,
transactions/walletTransactions/index.test.js:      activePeer: match.any,

■■■■[slaweet:~/git/hub-review/src/components] $
$ git grep 'peers:'
account/account.test.js:      peers: {
account/index.js:  peers: state.peers,
accountTransactions/index.test.js:    peers: peersReducer,
app/index.test.js:      peers: { data: { options: {} }, options: {} },
app/index.test.js:      peers: {
authenticate/index.js:  peers: state.peers,
header/header.test.js:    peers: { data: {}, options: {} },
header/index.js:  peers: state.peers,
header/index.test.js:    peers: { data: {}, options: {} },
loadingBar/index.js:  peers: state.peers,
loadingBar/loadingBar.test.js:    peers: {},
loadingBar/loadingBar.test.js:    wrapper.setProps({ peers: { data: {} } });
login/index.js:  peers: state.peers,
registerDelegate/index.js:  peers: state.peers,
singleTransaction/index.js:  peers: state.peers,
transactions/explorerTransactions/index.test.js:    peers: peersReducer,
transactions/transactionDetailView.js:  peers: state.peers,
transactions/transactionDetailView.test.js:        peers: {
transactions/transactionDetailView.test.js:        peers: {
transactions/transactionList.test.js:    peers: peersReducer,
transactions/transactionOverview.js:  peers: state.peers,
transactions/transactionOverview.test.js:      peers: {
transactions/transactionOverview.test.js:      peers: { status: {}, options: {}, data: {} },
transactions/transactionType.test.js:  peers: {
transactions/walletTransactions/index.test.js:    peers: peersReducer,
voteUrlProcessor/index.test.js:    peers: { data: {} },
voting/index.test.js:      peers: {},
voting/voting.test.js:    peers: peersReducer,
votingListView/votingHeader.test.js:    peers: {
```

**Pull request merged**